src/components/ResultsCount/_results-count.scss

```diff
@@ -9,7 +9,7 @@
 .cc-results-count {
   border-top: 1px solid var(--colour-grey-10);
   font-size: var(--body-md);
-  padding-bottom: var(--space-lg);
+  padding-bottom: calc(3 * var(--space-unit));
```

Comment:
Why have you changed this from space-lg to a fixed value? Should the spacing not be responsive?

Reply:
This is updated to match with this design: https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5fc78b10d77e9f0619dcb471